react-multiple-pages/my-app/src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import { Routes, Route } from 'react-router-dom';
+import { Header } from './components/Header';
+import { Dashboard } from './pages/Dashboard';
+import { Details } from './pages/Details';
+import { About } from './pages/About';
+import { NotFound } from './pages/NotFound';
+
+function fn() {}
+
+export function App() {
+  return (
+    <Routes>
+      <Route path="/" element={<Header></Header>}>
```

Comment:
No need to change this now but if you aren't going to have any child content between opening an closing Components tags you can just express as a self closing tag eg. `<Header />`